Java/PlatinumGenomes-variant-transformation/src/test/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentDataTest.java

```diff
@@ -63,17 +66,40 @@ public void testFilterVariantCallsFn() {
         ListValue.newBuilder()
             .addValues(Value.newBuilder().setStringValue("VQSRTrancheSNP99.90to100.00").build())
```

Comment:
This test case is just trying to show a non-passing filter value that might occur in data imported to the API.